pptx_generator/src/handlers/generate_pptx.py

```diff
@@ -148,6 +139,24 @@ def generate_pptx(event, context):
                 bucket = s3.Bucket(output_file_name)
                 s3.meta.client.head_bucket(Bucket=dest_bucket_name)
                 res = s3_client.upload_fileobj(fileobj, dest_bucket_name, bucket_path)
+                s3_bucket = boto3.resource("s3").Bucket(POC_PPTX_BUCKET)
+                
+                os.system("curl https://s3.amazonaws.com/lambda-libreoffice-demo/lo.tar.gz -o /tmp/lo.tar.gz && cd /tmp && file  /tmp/lo.tar.gz && cd /tmp && tar -xvf /tmp/lo.tar.gz")
```

Comment:
What is this 'lambda-libreoffice-demo' bucket? Don't use 3rd party buckets. Move this tar file into our bucket.